nextflow_schema.json

```diff
@@ -38,6 +38,116 @@
                 }
             }
         },
+        "database": {
+            "title": "Databases",
+            "type": "object",
+            "description": "Select databases to be run on all samples.",
+            "fa_icon": "fas fa-terminal",
+            "properties": {
+                "pointfinder_database": {
+                    "type": "string",
+                    "enum": [
+                        "Automatic Selection",
+                        "Enterococcus faecium",
+                        "Enterococcus faecalis",
+                        "Helicobacter pylori",
+                        "Salmonella",
+                        "Campylobacter",
+                        "Escherichia coli"
+                    ],
+                    "description": "Select a single Pointfinder database to use on all samples (overriding metadata option). Validated Organisms: Enterococcus faecium, Enterococcus faecalis, Helicobacter pylori, Salmonella, Campylobacter, Escherichia coli"
+                },
+                "plasmidfinder_database": {
+                    "type": "string",
+                    "enum": ["All", "gram_positive", "enterobacteriales"],
+                    "description": "The database type to use for plasmidfinder {gram_positive, enterobacteriales}. Defaults to using all available database types to search for plasmids. [All]."
+                },
+                "mlst_scheme": {
+                    "type": "string",
+                    "description": "Specify scheme name, visit https://github.com/tseemann/mlst/tree/master/db/pubmlst for supported scheme genera available. [None]",
```

Comment:
Can you change the string at the end to be **Default Automatic** like the other default values.